pnpm-lock.yaml

```diff
@@ -2869,12 +2884,23 @@ importers:
   components/tally:
     specifiers: {}
 
+  components/taxjar:
+    specifiers: {}
+
   components/td_ameritrade:
     specifiers: {}
 
   components/teamdeck:
     specifiers: {}
 
+  components/teamgate:
+    specifiers:
+      '@pipedream/platform': ^0.10.0
+      moment: ^2.29.4
+    dependencies:
+      '@pipedream/platform': 0.10.0
+      moment: 2.29.4
+
```

Comment:
As I see, there is no dependencies for Teamgate app on here and on master branch. Can we remove it if not needed?